utils/serviceRequest.js

```diff
@@ -103,11 +104,14 @@ class ABServiceRequest extends ServiceCote {
       } catch (err) {
          err._serviceRequest = key;
          err._params = paramStack;
-         this.req.notify.developer(err, {
-            message: `Could not request (${key}) - ${JSON.stringify(
-               paramStack
-            )}`,
-         });
+         if (key != "log_manager.notification") {
+            // If there was an error requesting log_manger.notification, don't use req.notify or we can get caught in an infinite loop
+            this.req.notify.developer(err, {
+               message: `Could not request (${key}) - ${JSON.stringify(
+                  paramStack
+               )}`,
+            });
+         }
```

Comment:
so what is the right thing to do if the `key == "log_manager.notification"` ?

should we at least `this.req.log()` the error?

Reply:
`notify()` does a `req.log` if it gets an error back:
```
this.req.serviceRequest("log_manager.notification", jobData, (err) => {
   this.req.log(err);
});
```
is that enough, or should we add here as well?